alpharaw/thermo.py

```diff
@@ -26,8 +26,7 @@
     "faims_cv",
 ]
 
-#: The auxiliary items and types that can be accessed from thermo RawFileReader.
-__auxiliary_item_dtypes__ = {
+auxiliary_item_dtypes: dict = {
```

Comment:
I've never used this so far. Is this used for type hints and in the docs?

Reply:
This is used to control the dtypes of each entry, and for the docs of course.